common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -536,10 +529,7 @@
                         } else {
                             if (state.isTouch) {
                                 self.subtitlesEl.find('.subtitles-menu').html(
-                                    gettext(
-                                        '<li>Transcript will be displayed when ' +
-                                        'you start playing the video.</li>'
-                                    )
+                                    gettext('<li>Transcript will be displayed when you start playing the video.</li>')
```

Comment:
The html shouldn't be included as part of the translation.   A translator may not know what it is and it may not be kept, or could be mistranslated.